opentelemetry-sdk/src/opentelemetry/sdk/_metrics/point.py

```diff
@@ -51,6 +68,16 @@ class Histogram:
     sum: Union[int, float]
     aggregation_temporality: AggregationTemporality
 
+    def _dict(self):
```

Comment:
I think you could just use [`asdict`](https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict) here?

Reply:
Good point!